dags.py

```diff
@@ -0,0 +1,64 @@
+from airflow import DAG
+from airflow.utils.dates import days_ago
+from airflow.operators.python_operator import PythonOperator
+from airflow.operators.bash_operator import BashOperator
+
+from main_airflow import main
+from data_access import model_from_to_json
+from configs import learning_model_path
+
+
+
+default_args = {
+    "owner": 'airflow',
+    "start_date": days_ago(1)
+}
+
+# running event base
+dag = DAG(dag_id='anomaly_detection_framework_fe_train_predict_dashboard',
+          description="Simple test6 DAG",
+          default_args=default_args,
+          schedule_interval='@once')
+
```

Comment:
This dag works only once with pipe line feature engineering, train models, rediction of models, how dashboard